app/views/layouts/application.html.erb

```diff
@@ -17,8 +17,14 @@
         <div class="menu-item">
           <%= menu_link_to 'All users', users_path %>
         </div>
+        <% if user_signed_in? %>
+          <div class="menu-item">
+            <%= menu_link_to 'Pending friends', pending_friends_path %>
+          </div>
+        <% end %>
       </div>
       <% if current_user %>
+        <%= current_user.name %>
         <%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
       <% else %>
         <%= link_to 'Sign in', user_session_path %>
```

Comment:
- [ ] View-related logic should be encapsulated in View helpers (app/helpers/*_helper.rb)

Reply:
Done